### DISCUSSION TITLE
Make vikas collab

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [collab-community/journey-book#38] [General]
@Adarsh-jaiss Please make @vikasganiga05 a project collab

**Comment by Adarsh-jaiss** [comment]
Okay

**Reply by Adarsh-jaiss** [reply]
Done 👍

**Reply by KendallDoesCoding** [reply]
Thanks

**Comment by KendallDoesCoding** [comment]
@vikasganiga05 Please accept the invitation.

**Reply by vikasganiga05** [reply]
> @vikasganiga05 Please accept the invitation.

Done :+1: Thanks!
